sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/StateStoreSuite.scala

```diff
@@ -978,6 +978,28 @@ abstract class StateStoreSuiteBase[ProviderClass <: StateStoreProvider]
     assert(store.metrics.memoryUsedBytes > noDataMemoryUsed)
   }
 
+  test("SPARK-34270: StateStoreMetrics.combine should not override individual metrics") {
+    val customMetric = StateStoreCustomSumMetric("metric1", "custom metric 1")
```

Comment:
Thank you for updating. Maybe, `customMetric` -> `customSumMetric`?

Reply:
Changed.